.zuul.yaml

```diff
@@ -0,0 +1,232 @@
+
+- job:
+    name: stf-crc-local_build
+    parent: stf-base
+    abstract: true
+    description: |
+      Build images locally and deploy STF
+    vars:
+      scenario: "local_build"
+
+- job:
+    name: stf-crc-local_build-index_deploy
+    parent: stf-base
+    abstract: true
+    description: |
+      Build STF locally and deploy from index
+    vars:
+      scenario: "local_build-index_deploy"
+
+- job:
+    name: stf-crc-ocp_412-nightly_bundles
+    parent: stf-crc-nightly_bundles
+    description: |
+      Deploy STF using the nightly bundles on OCP 4.12
+    nodeset: stf-crc_extracted-ocp412
+
+- job:
+    name: stf-crc-ocp_413-nightly_bundles
+    parent: stf-crc-nightly_bundles
+    description: |
+      Deploy STF using the nightly bundles on OCP 4.13
+    nodeset: stf-crc_extracted-ocp413
+
+- job:
+    name: stf-crc-ocp_414-nightly_bundles
+    parent: stf-crc-nightly_bundles
+    description: |
+      Deploy STF using the nightly bundles on OCP 4.14
+    nodeset: stf-crc_extracted-ocp414
+
+- job:
+    name: stf-crc-ocp_412-local_build
+    parent: stf-crc-local_build
+    description: |
+      Build images locally and deploy STF on OCP 4.12
+    nodeset: stf-crc_extracted-ocp412
+
+- job:
+    name: stf-crc-ocp_413-local_build
+    parent: stf-crc-local_build
+    description: |
+      Build images locally and deploy STF on OCP 4.13
+    nodeset: stf-crc_extracted-ocp413
+
+- job:
+    name: stf-crc-ocp_414-local_build
+    parent: stf-crc-local_build
+    description: |
+      Build images locally and deploy STF on OCP 4.14
+    nodeset: stf-crc_extracted-ocp414
+
+- job:
+    name: stf-crc-ocp_412-local_build-index_deploy
+    parent: stf-crc-local_build-index_deploy
+    description: |
+      Build STF locally and deploy from index on OCP 4.12
+    nodeset: stf-crc_extracted-ocp412
+
+- job:
+    name: stf-crc-ocp_413-local_build-index_deploy
+    parent: stf-crc-local_build-index_deploy
+    description: |
+      Build STF locally and deploy from index on OCP 4.13
+    nodeset: stf-crc_extracted-ocp413
+
+- job:
+    name: stf-crc-ocp_414-local_build-index_deploy
+    parent: stf-crc-local_build-index_deploy
+    description: |
+      Build STF locally and deploy from index on OCP 4.14
+    nodeset: stf-crc_extracted-ocp414
+
+- job:
+    name: stf-crc-ocp_412-local_build-index_deploy
+    parent: stf-crc-local_build-index_deploy
+    description: |
+      Build STF locally and deploy from index on OCP 4.12
+    vars:
+      crc_ocp_bundle: 'https://mirror.openshift.com/pub/openshift-v4/clients/crc/bundles/openshift/4.12.13/crc_libvirt_4.12.13_amd64.crcbundle'
+
+- job:
+    name: stf-crc-ocp_413-local_build-index_deploy
+    parent: stf-crc-local_build-index_deploy
+    description: |
+      Build STF locally and deploy from index on OCP 4.13
+    vars:
+      crc_ocp_bundle: 'https://mirror.openshift.com/pub/openshift-v4/clients/crc/bundles/openshift/4.13.14/crc_libvirt_4.13.14_amd64.crcbundle'
```

Comment:
These should be removed, since they are superceeded by the jobs in L174 and L181.

```suggestion
```